docs/runbooks/PromscaleIngestHighDataDuplication.md

```diff
@@ -0,0 +1,19 @@
+# PromscaleIngestHighDataDuplication
+
+## Meaning
+
+Client payload has either duplicates or retrying many a time for the data which has been already ingested.
+
+
+## Impact
+
+Ingestion performance will be poor
+
+
+## Diagnosis
+1. If Prometheus is running in HA mode, go to [Prometheus high availability](#prometheus-high-availability)
```

Comment:
WDYT about an specific entry in diagnosis for prometheus HA running without the labels:

When running a Prometheus HA deployment there's an increase in ingest due to duplicate  data being sent to Promscale. We can view the ingest of duplicates with:

```
sum by(job, instance, type) (
          rate(promscale_ingest_duplicates_total{kind="sample"}[5m])
      )
```. 

This could happen if the Prometheus HA deployment is not configured to decorate the samples with the metadata from the replica that's pushing the data. In this scenario, two or more Prometheus replicas from the same cluster will be sending the exact same datapoints, and since there's no cluster/replica metadata, Promscale doesn't have the information needed to just accept the data from one of them and will try to persist them all.